### ISSUE TITLE
It's unclear that discussions in modding v2 have a border when clicked on

### ISSUE THREAD

**Issue by Xinnoh** [ppy/osu-web#2922]
Clicking on a discussion creates a small shadow behind it. Clicking a permalink to a discussion also highlights it, useful for finding the discussion of the link you clicked.

I had no idea this feature existed until 5 minutes ago, the highlight behind a discussion should be more prominent, maybe a colour or something

**Comment by peppy** [comment]
I agree this could be made a little clearer, but it may also already be fixed in the new design.
